cli/data.py

```diff
@@ -66,6 +66,37 @@ def main():
     pass
 
 
+def _run_dataset_filters(
```

Comment:
Good enough for now. Longer term I'd rather have a way to keep a copy of before and after the filters, than be able to see what changed at those points of the pipeline as input data and/or code changes.

Reply:
👍 i could imagine a world where this is 3 steps with separate outputs - combine, filter, aggregate